### PR TITLE
unnecessary await

### PR DESCRIPTION
**Pull request by a14n**

### CODE REVIEW

File: packages/flutter_driver/lib/src/driver/driver.dart [hunk 1]
```diff
@@ -342,7 +342,7 @@ class FlutterDriver {
     Future<Health> checkHealth() async {
       try {
         // At this point the service extension must be installed. Verify it.
-        return await driver.checkHealth();
+        return driver.checkHealth();
```
Comment (a14n) [inline packages/flutter_driver/lib/src/driver/driver.dart]: humm, the `await` removal is not correct here in a try-catch section. I need to update the lint.

**Comment by Hixie** [comment]
What makes these "unnecessary"?

**Comment by a14n** [comment]
"unnecessary" because they are in a `async` body and the expression is alreary a Future. You can see some cases in https://github.com/dart-lang/linter/blob/master/test/rules/unnecessary_await_in_return.dart

**Review by jonahwilliams — APPROVED** [review]
LGTM

**Comment by a14n** [comment]
According to this comment https://github.com/flutter/flutter/pull/25922#discussion_r244614379 I'm going to close this PR unless @Hixie's opinion has changed.

**Pull request closed without merging**